Makefile

```diff
@@ -29,7 +29,11 @@ check-go-version:
 ESBUILD_RACE ?= -race
 
 test-go:
+ifeq (s390x,$(shell uname -p))
+	go test ./internal/...
+else
```

Comment:
Why is this necessary?

Reply:
`-race` option is not supported on s390x arch. If we don't remove this flag for s390x, we get following error:
```
go test: -race is only supported on linux/amd64, linux/ppc64le, linux/arm64, freebsd/amd64, netbsd/amd64, darwin/amd64, darwin/arm64, and windows/amd64
```

Reply:
Ok, good to know. I'll disable it by default in the future then.